index.md

```diff
@@ -2,15 +2,16 @@
 layout: page
 title: Flutter
 ---
-Flutter is a new way to build high-performance, cross-platform mobile apps.
-Flutter is optimized for today's, and tomorrow's, mobile devices. We are focused
-on low-latency input and high frame rates on Android and iOS.
+
+Flutter is a new project to help developers
+build high-performance, high-fidelity,
+cross-platform mobile apps.
```

Comment:
Ah, good point!

I was trying to simplify the tag line here. Open to suggestions. (e.g. we should add "iOS and Android" back in), but would like to try to keep it down to one sentence.

I'm happy to update everywhere if you like the new phrasing with the one addition.